packages/cli-platform-android/src/commands/runAndroid/index.ts

```diff
@@ -42,6 +56,50 @@ export type AndroidProject = NonNullable<Config['project']['android']>;
 async function runAndroid(_argv: Array<string>, config: Config, args: Flags) {
   link.setPlatform('android');
 
+  let {packager, port} = args;
+
+  const packagerStatus = await isPackagerRunning(port);
+
+  const handlePortUnavailable = async () => {
```

Comment:
`handlePortUnavailable` seems to be duplicated with minimal changes across 3 packages. Let's see if we can refactor this

Reply:
done 👍